controllers/drplacementcontrol.go

```diff
@@ -296,7 +335,7 @@ func (d *DRPCInstance) RunFailover() (bool, error) {
 	// NOTE: If an initial spec started with the failover action, it will be failed over to the
 	// provided failover cluster. This is an inadvertent outcome, but deemed not an issue.
 	failoverClusterVRG, ok := d.vrgs[d.instance.Spec.FailoverCluster]
-	if !ok {
+	if !ok || d.isVRGSecondary(failoverClusterVRG) {
```

Comment:
Line 345 should not exist.  It was left there by mistake after we moved it to line 338.  Testing passed because for both cases it works.  In other words, if it is volsync, the VRG will be there, if not we will create it.  If it is VolRep, the VRG will not exist on the failover cluster.  But, as I said, the block starting from line 345 will be removed.